modular_skyrat/modules/contractor/code/datums/contract.dm

```diff
@@ -0,0 +1,249 @@
+	// method of healing them, consequence free, to a reasonable amount of health.
+	target.reagents.add_reagent(/datum/reagent/medicine/omnizine, 20)
+
+	target.flash_act()
+	target.add_confusion(10)
+	target.blur_eyes(5)
+	to_chat(target, span_warning("You feel strange..."))
+	sleep(6 SECONDS)
+	to_chat(target, span_warning("That pod did something to you..."))
+	target.Dizzy(35)
+	sleep(6 SECONDS)
+	to_chat(target, span_warning("Your head pounds... It feels like it's going to burst out your skull!"))
+	target.flash_act()
+	target.add_confusion(20)
+	target.blur_eyes(3)
+	sleep(3 SECONDS)
+	to_chat(target, span_warning("Your head pounds..."))
+	sleep(10 SECONDS)
+	target.flash_act()
+	target.Unconscious(200)
+	to_chat(target, "<span class='reallybig hypnophrase'>A million voices echo in your head... <i>\"Your mind held many valuable secrets - \
+				we thank you for providing them. Your value is expended, and you will be ransomed back to your station. We always get paid, \
+				so it's only a matter of time before we ship you back...\"</i></span>")
+	target.blur_eyes(10)
+	target.Dizzy(15)
+	target.add_confusion(20)
+
+// We're returning the victim
+/datum/syndicate_contract/proc/returnVictim(mob/living/target)
+	var/list/possible_drop_loc = list()
+
+	for(var/turf/possible_drop in contract.dropoff.contents)
+		if(!(!isspaceturf(possible_drop) && !isclosedturf(possible_drop)))
+			continue
+		if(!possible_drop.is_blocked_turf())
+			possible_drop_loc.Add(possible_drop)
+
+	if (length(possible_drop_loc) > 0)
+		var/pod_rand_loc = rand(1, possible_drop_loc.len)
+
+		var/obj/structure/closet/supplypod/return_pod = new()
+		return_pod.bluespace = TRUE
+		return_pod.explosionSize = list(0,0,0,0)
+		return_pod.style = STYLE_SYNDICATE
+
+		do_sparks(8, FALSE, target)
+		target.visible_message(span_notice("[target] vanishes..."))
+
+		for(var/obj/item/target_item in target)
+			if(ishuman(target))
+				var/mob/living/carbon/human/human_target = target
+				if(target_item == human_target.w_uniform)
+					continue //So all they're left with are shoes and uniform.
+				if(target_item == human_target.shoes)
+					continue
+			target.dropItemToGround(target_item)
+
+		for(var/obj/item/target_item in victim_belongings)
+			target_item.forceMove(return_pod)
+
+		target.forceMove(return_pod)
+
+		target.flash_act()
+		target.blur_eyes(30)
+		target.Dizzy(35)
+		target.add_confusion(20)
+
+		new /obj/effect/pod_landingzone(possible_drop_loc[pod_rand_loc], return_pod)
+	else
+		to_chat(target, "<span class='reallybig hypnophrase'>A million voices echo in your head... <i>\"Seems where you got sent here from won't \
+					be able to handle our pod... You will die here instead.\"</i></span>")
+		if(!iscarbon(target))
+			return
+		var/mob/living/carbon/unlucky_fellow = target
+		if (unlucky_fellow.can_heartattack())
+			unlucky_fellow.set_heartattack(TRUE)
```

Comment:
What happens if this happens, but the victim has muscled veins or something?

Reply:
This was inherited code from how contractor did it before, it's pretty awful and I'm going to redo it